examples/exploring_priors.jl

```diff
@@ -0,0 +1,105 @@
+# # Exploring priors
+#
+# This example explores specifying priors using OceanTurbulenceParameterEstimation. 
+#
+# ## Install dependencies
+#
+# First let's make sure we have all required packages installed.
+
+# ```julia
+# using Pkg
+# pkg"add OceanTurbulenceParameterEstimation, Oceananigans, Distributions, CairoMakie"
+# ```
+
+# First we load few things
+
+using OceanTurbulenceParameterEstimation
+using CairoMakie
+using Distributions
+
+# `OceanTurbulenceParameterEstimation` supports three types of prior
+# distributions. The normal prior distribution,
+
+normal_prior = Normal(0.1, 0.1)
```

Comment:
how about we add a method to `Normal` like:

```
import Distributions: Normal

Normal(; mean, std) = Normal(mean, std)
```

so we can call it with the bit-more-meaningful

```
normal_prior = Normal(mean=0.1, std=0.1)
```
?

Reply:
We can't pirate the name `Normal` but we could write a constructor `normal`

Reply:
The only reason I didn't do that is because it doesn't do any calculations, so its not need for practical reasons, just aeshtetics...

Reply:
ok!